src/components/LandmarkDrawer.tsx

```diff
@@ -101,11 +120,12 @@ function getAssessmentDate(assessment: LandmarkAssessment) {
 function LandmarkAssessmentHistory({ landmark } : {
   landmark: Landmark;
 }) {  
-  const [user] = useUserContext();
+  const [user] = useUserContext();      
```

Comment:
spaces. please make sure you always self review your PRs before sending me for review.